spring-cloud-aws-autoconfigure/src/main/java/io/awspring/cloud/autoconfigure/dynamodb/DynamoDbClientAutoConfiguration.java

```diff
@@ -30,102 +29,107 @@
 import org.springframework.boot.autoconfigure.condition.ConditionalOnMissingBean;
 import org.springframework.boot.autoconfigure.condition.ConditionalOnProperty;
 import org.springframework.boot.autoconfigure.condition.NoneNestedConditions;
-import org.springframework.boot.context.properties.EnableConfigurationProperties;
 import org.springframework.boot.context.properties.PropertyMapper;
 import org.springframework.context.annotation.Bean;
 import org.springframework.context.annotation.Conditional;
 import org.springframework.context.annotation.Configuration;
 import software.amazon.awssdk.auth.credentials.AwsCredentialsProvider;
-import software.amazon.awssdk.enhanced.dynamodb.DynamoDbEnhancedClient;
 import software.amazon.awssdk.regions.providers.AwsRegionProvider;
+import software.amazon.awssdk.services.dynamodb.DynamoDbAsyncClient;
+import software.amazon.awssdk.services.dynamodb.DynamoDbAsyncClientBuilder;
 import software.amazon.awssdk.services.dynamodb.DynamoDbClient;
 import software.amazon.awssdk.services.dynamodb.DynamoDbClientBuilder;
+import software.amazon.dax.ClusterDaxAsyncClient;
 import software.amazon.dax.ClusterDaxClient;
 
 /**
- * {@link EnableAutoConfiguration Auto-configuration} for DynamoDB integration.
+ * {@link DynamoDbClientAutoConfiguration Auto-configuration} for DynamoDB integration.
```

Comment:
`EnableAutoConfiguration` was correct here. This is the convention used in spring boot for autoconfigurations Javadocs.

Since now DynamoDb autoconfiguration is broken into multiple autoconfigurations, Javadocs needs to be more detailed though

Reply:
What do you suggest?

Reply:
We should add inner static classes inside of the `DynamoDbClientAutoConfiguration.class` and return the naming.